packages/nimble-blazor/createPlaywrightVersionPropertiesFile.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
```

Comment:
Can we move these under the build folder? (I know it's not really build but it's more like an automation dumping ground folder) ie:
```
/build/generate-playwright-version-properties/
  /source/index.js
  /source/Playwright.PackageVersion.template
  /dist/<generate the props file here>
```

Reply:
Updated